code401challenges/Data-Structures/src/main/java/Data/Structures/LinkedList/LinkedList.java

```diff
@@ -0,0 +1,43 @@
+package Data.Structures.LinkedList;
+
+
+import org.w3c.dom.Node;
```

Comment:
this import is importing from outside the project while you want your node. You probably auto completed line 7, when you first used "Node" before defining the class Node within your own project. 
Simply delete this line of code and you will move past the first error.